subxt/Cargo.toml

```diff
@@ -90,3 +106,15 @@ sp-runtime = { workspace = true }
 sp-keyring = { workspace = true }
 sp-version = { workspace = true }
 assert_matches = { workspace = true }
+tracing-subscriber = { workspace = true }
```

Comment:
Note: this is really useful for making sure the light-client is working (ie GrandPa warp sync)

Reply:
you mean by getting the logs? 

Is there a specific smoldot log target/level that is useful for this? 
Could be useful with a comment in the example code I guess..

Reply:
The logs look similar to:

```bash
2023-06-16T11:33:36.099647Z  INFO smoldot: Chain initialization complete for dev. Name: "Development". Genesis hash: 0x4e1e…05c2. State root hash: 0x50e69d13db00a26a2834fa5ed6d6f2d99c5b965a6170da09b682c2e817f105ae. Network identity: 12D3KooWFmJ4hSvYaYTVDnAmPvDgH775XG67XkwxfbAAALKGw8CY. Chain specification or database starting at: 0x4e1e…05c2 (#0)

2023-06-16T11:33:36.100055Z  WARN json-rpc-dev: The JSON-RPC client has just called a JSON-RPC function from the legacy JSON-RPC API (chain_getBlockHash). Legacy JSON-RPC functions have loose semantics and cannot be properly implemented on a light client. You are encouraged to use the new JSON-RPC API <https://github.com/paritytech/json-rpc-interface-spec/> instead. The legacy JSON-RPC API functions will be deprecated and removed in the distant future.

2023-06-16T11:33:37.439069Z  INFO sync-service-dev: GrandPa warp sync finished to #68 (0x4f1b…3651)
2023-06-16T11:33:37.439437Z  INFO runtime-dev: Finalized block runtime ready. Spec version: 9390. Size of `:code`: 1.4 MiB.

2023-06-16T11:33:38.681854Z  INFO tx-service-dev: Successfully validated transaction 0x8f19…4c46
```

The `INFO` works fine for use, we care about the `INFO sync-service-dev: GrandPa warp sync finished to #68 (0x4f1b…3651)` and `INFO runtime-dev: Finalized block runtime ready. Spec version: 9390. Size of `:code`: 1.4 MiB.` with which I had some problems in the past :D